met-web/src/components/feedback/listing.tsx

```diff
@@ -0,0 +1,162 @@
+        label: '',
+    },
+};
+
+const FeedbackListing = () => {
+    const [feedbacks, setFeedbacks] = useState<Feedback[]>([]);
+    const [paginationOptions, setPaginationOptions] = useState<PaginationOptions<Feedback>>({
+        page: 1,
+        size: 10,
+        sort_key: 'rating',
+        nested_sort_key: null,
+        sort_order: 'asc',
+    });
+    const [pageInfo, setPageInfo] = useState<PageInfo>(createDefaultPageInfo());
+    const [tableLoading, setTableLoading] = useState(true);
+    const dispatch = useAppDispatch();
+
+    const { page, size, sort_key, nested_sort_key, sort_order } = paginationOptions;
+
+    useEffect(() => {
+        loadFeedbacks();
+    }, [paginationOptions]);
+
+    const loadFeedbacks = async () => {
+        try {
+            setTableLoading(true);
+            const response = await getFeedbacksPage({
+                page,
+                size,
+                sort_key: nested_sort_key || sort_key,
+                sort_order,
+            });
+            setFeedbacks(response.items);
+            setPageInfo({
+                total: feedbacks.length,
+            });
+            setTableLoading(false);
+        } catch (error) {
+            console.log(error);
+            dispatch(
+                openNotification({
+                    severity: 'error',
+                    text: 'Error occurred while trying to fetch feedbacks, please refresh the page or try again at a later time',
+                }),
+            );
+            setTableLoading(false);
+        }
+    };
+
+    const headCells: HeadCell<Feedback>[] = [
+        {
+            key: 'rating',
+            numeric: true,
+            disablePadding: true,
+            label: 'Feedback',
+            allowSort: true,
+            getValue: (row: Feedback) => customRatings[row.rating].icon,
+        },
+        {
+            key: 'comment_type',
+            numeric: false,
+            disablePadding: false,
+            label: 'Feedback Type',
+            allowSort: true,
+            getValue: (row: Feedback) => row.comment_type,
+        },
+        {
+            key: 'comment',
+            numeric: true,
+            disablePadding: false,
+            label: 'Message',
+            allowSort: true,
+            getValue: (row: Feedback) => row.comment,
+        },
+    ];
+
+    return (
+        <MetPageGridContainer
+            direction="row"
+            justifyContent="flex-start"
+            alignItems="flex-start"
+            container
+            columnSpacing={2}
+            rowSpacing={1}
+        >
+            <Grid item xs={12} lg={10}>
+                <Stack
+                    direction={{ xs: 'column', md: 'row' }}
+                    spacing={1}
+                    width="100%"
+                    justifyContent="flex-end"
+                    sx={{ p: 2 }}
+                >
+                    <PrimaryButton component={Link} to="/feedback" data-testid="feedback">
```

Comment:
I now marie anik has this button in her design but honesly I don't know what this is about, I say we drop it for now since it has no functionality (only takes it to the same page)

Reply:
ok ill remove it